targets/TARGET_STM/TARGET_STM32F1/TARGET_BLUEPILL_F103C8/device/stm32f1xx_hal_msp.c

```diff
@@ -0,0 +1,127 @@
+/**
```

Comment:
Same here - if really needed this file stm32f1xx_hal_msp.c belongs to application layer.